README.md

```diff
@@ -95,7 +95,7 @@ X/Y/Z are integers. A and B are arbitrary Expressions.
 | `dYkZ` `XdYkZ`    | Keep dice notation. Keeps the *highest* Z dice from the result of `XdY`. Alias for `XdYkhZ`. May be used with `ns` and `!`.                              |
 | `dYkZ` `XdYkZ`    | Keep dice notation. Keeps the *highest* Z dice from the result of `XdY`. May be used with `ns` and `!`.                                                  |
 | `dYklZ` `XdYklZ`  | Keep dice notation. Keeps the *lowest* Z dice from the result of `XdY`. May be used with `ns` and `!`.                                                   |
-
+| 'dYcZ 'XdYcZ'     | Critrange notation. Bolds all rolls greater than or equal to Z. Cannot be used with keep or drop notations.                                              |
```

Comment:
These have to be backticks (`) rather than quotes (') to get the effect in markdown.

Reply:
Woops! I didn't look closely enough at the previous ones!